src/main/java/uk/gov/hmcts/reform/userprofileapi/config/LaunchDarklyConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.userprofileapi.config;
+
+import com.launchdarkly.sdk.server.LDClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import uk.gov.hmcts.reform.userprofileapi.util.FeatureConditionEvaluation;
+
+@Configuration
+public class LaunchDarklyConfiguration implements WebMvcConfigurer {
+
+    @Bean
+    public LDClient ldClient(@Value("${launchdarkly.sdk.key}") String sdkKey) {
+        return new LDClient(sdkKey);
+    }
+
+    @Autowired
+    private FeatureConditionEvaluation featureConditionEvaluation;
+
+    @Override
+    public void addInterceptors(InterceptorRegistry registry) {
+        registry.addInterceptor(featureConditionEvaluation)
+                .addPathPatterns(
+                        "/v1/userprofile/users**");
+    }
```

Comment:
We might need to be specific in term of the pattern in future so that there are no conflicts.

Reply:
Yes, if any other API is built with `/v1/userprofile/users` as the path then we will need to be specific as you said